service/dap/server.go

```diff
@@ -1684,10 +1757,10 @@ func (s *Server) doCommand(command string) {
 
 		// TODO(polina): according to the spec, the extra 'text' is supposed to show up in the UI (e.g. on hover),
```

Comment:
Does text show up in some new way with the new changes? Do we need it since you do the same thing in exception info? This todo likely can go away now, right? You could turn this into just a comment explaining that we had to use exception info because text did not work with multiple goroutines.